src/pages/Room/Room.js

```diff
@@ -62,13 +69,10 @@ class Room extends Component {
   };
 
   handleShow = () => {
-    this.setState(prevState => ({
-      show: true
-    }));
-    this.socket.emit('show', true);
+    this.socket.emit('show');
```

Comment:
Because we have two actions, `Show` and `Clear`. There is no need to pass `true` for `Show` action.

If we will abstract the two events into one, we can use a boolean that time.